azure_pipelines/templates/tests-with-conf-file.yml

```diff
@@ -0,0 +1,73 @@
+parameters:
+  schema: 'MSAL Test Automation (iOS)'
+  build: 'MSAL Test Automation (iOS)'
+
+steps:
+  - checkout: self
+    clean: true
+    submodules: true
+    fetchDepth: 1
+    persistCredentials: true
+    path: s
+    
+  - task: Bash@3
+    displayName: 'Select Xcode version'
```

Comment:
We use different Xcode versions in this file and in `pr-validation.yml`. 

What do you think about creating a new `parameter` here and in `pr-validation.yml`, so we set the same version in both files?

Ideally we would have only one single place where to put all these variables, but I don't know if that's possible.

Reply:
Yeah, I removed the macOS declaration for all jobs because the pipeline was failing (it was selecting ubuntu as image instead of mac). Here is the commit:
https://github.com/AzureAD/microsoft-authentication-library-for-objc/pull/2236/commits/21cde46e9f1eda51eef3952e9b91b4f4b52bb333
I used the same approach as the automation.yml file, but it didn't work

Reply:
Are you suggesting creating a new parameter to set different xcode version from both script?

Reply:
Resolving this comment as discussed internally.

Reply:
Sharing here too: variables can't be created in a steps templates, but only in variables template